next/tailwind.config.js

```diff
@@ -77,16 +77,30 @@ module.exports = {
         "270": "270deg"
       },
       fontSize: {
-        xxs: ['12px', '18px'],
-        xs: ['12px', '18px'],
-        sm: ['16px', '24px'],
-        default: ['20px', '30px'],
-        md: ['24px', '36px'],
-        lg: ['32px', '41.6px'],
-        xl: ['36px', '42.3px'],
-        '2xl': ['40px', '52px'],
-        xxxl: ['48px', '62.4px'],
+        'p-xs': ['12px', '18px'],
```

Comment:
a couple changes proposed with this config - these are the defaults available in tailwind https://tailwindcss.com/docs/font-size#setting-the-font-size , I'd like to prevent someone using one of these options by accident

if you move this config one level higher out of the ''extends" object, it will replace instead of extending. Of couse, this unfortunately requires the hunting down of the default tailwind classes for font sizes but that shouldn't be too bad.

The other suggestion is to align this a bit more with the tailwind approach - that is, use `p-base` and `h-base` instead of p-default and to use 3xl instead of xxxl.